notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb

```diff
@@ -665,9 +624,9 @@
         "id": "79eaa73a"
       },
       "source": [
-        "### Import to BQ\n",
+        "### Import to BigQuery\n",
         "\n",
-        "This component takes the csv file and imports it to a table in BigQuery. If the dataset does not exist, it will be created. If a table with the same name already exists, it will be deleted and recreated"
+        "Firstly, define a component that loads the csv file and imports it to a BigQuery table. If the dataset doesn't exist, it gets created. If a table already exists with the same name, it gets overwritten."
```

Comment:
- change "Firstly" to "First"
- change "gets" to "it's"
     -  If the dataset doesn't exist, it's created
- delete "gets":
    - If a table already exists with the same name, it's overwritten